mycroft/util/__init__.py

```diff
@@ -109,5 +111,21 @@ def connected(host="8.8.8.8", port=53, timeout=3):
             return False
 
 
+def create_signal(signalName):
```

Comment:
Sorry I wasn't more explicit. The existing messagebus clients [all use an instance of event emitter](https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/messagebus/client/ws.py#L46) under the covers, and it should be pretty easy to expose the 'once' functionality.

I'm curious, why do you think the signal state needs to be persisted like this? Could you instead have something like this, assuming there's no naming collisions?
```
ws.emit(Message(signal_name))
```